common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -94,7 +95,32 @@ def save_subs_to_store(subs, subs_id, item, language='en'):
     return save_to_store(filedata, filename, 'application/json', item.location)
 
 
-def get_transcripts_from_youtube(youtube_id, settings, i18n):
+def transcript_name_youtube_video(youtube_text_api):
```

Comment:
You mean the transcript name or the youtube response? I don't have strong opinions either way